meeting_minutes/220728_SARIF_TC_63.md

```diff
@@ -0,0 +1,137 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+|            |           |                   |                         |
+Jeff Williams, Mary Martin, David Keaton, Thanassis Avgerinos, Pavel Mikula, Paul Anderson, Yekaterina O'Neil, Chris Wysopal, Nathan Baird, Michael Fanning, Aditya Sharad, Jerry Sullivan (MicroFocus) attending.
```

Comment:
All were voting except Aditya and Jerry.  (Aditya gained voting privileges at the end of the meeting, as correctly recorded in section 1.7.2.)